apps/epic-react/src/components/app/navigation/index.tsx

```diff
@@ -0,0 +1,342 @@
+              cx('flex items-start', {
+                'flex-col': isTablet,
+                'flex-row items-center': !isTablet,
+              }),
+            )}
+          >
+            {/* TODO: <Feedback /> component */}
+            {isAuthenticated ? (
+              <Feedback>{isTablet ? 'Send Feedback' : <Message />}</Feedback>
+            ) : null}
+
+            <ThemeToggle />
+
+            {isAuthenticated ? (
+              <TooltipProvider>
+                <Tooltip>
+                  <TooltipTrigger asChild>
+                    <Button
+                      variant="ghost"
+                      size="icon"
+                      onClick={() => signOut()}
+                      className={twMerge(
+                        cx(
+                          'w-auto border-none p-2 px-3 text-base text-text transition-opacity duration-150 ease-in-out hover:bg-transparent hover:opacity-100',
+                          {
+                            'opacity-100': isTablet,
+                            'opacity-75': !isTablet,
+                          },
+                        ),
+                      )}
+                    >
+                      {isTablet ? 'Log Out' : <Logout />}
+                      <span className="sr-only">Log out</span>
+                    </Button>
+                  </TooltipTrigger>
+                  {!isTablet && (
+                    <TooltipContent>
+                      <p>Log Out</p>
+                    </TooltipContent>
+                  )}
+                </Tooltip>
+              </TooltipProvider>
+            ) : null}
+          </div>
+        </>
+      )
+    }
+    return (
+      <div className="grid grid-flow-col items-center gap-5 text-xs sm:pr-2 sm:text-sm">
+        {!isTablet || !isMobile ? <Skeleton className="h-6 w-16" /> : null}
+        <Skeleton className="h-6 w-6" />
+      </div>
+    )
+  }
+
+  return hasMounted ? (
+    <header className="fixed top-0 z-30 mx-auto w-full print:hidden">
+      {showDiscountBar && (
+        <motion.div style={{y: dismissMessageBar}}>
+          <MessageBar
+            percentageDiscount={(
+              Number(currentSale.percentageDiscount) * 100
+            ).toString()}
+          />
+        </motion.div>
+      )}
+      <motion.nav
+        style={{y: showDiscountBar ? dismissMessageBar : 0}}
+        className="navigation flex w-full items-center justify-between border-b border-er-gray-100 px-2 py-2 shadow-sm"
+      >
+        <div className="flex flex-grow items-center">
+          <Link
+            href={
+              isAuthenticated &&
+              !isEmpty(sessionData?.user?.purchases) &&
+              !purchasedOnlyTeam
+                ? '/learn'
+                : '/'
+            }
+            className="flex items-center"
+          >
+            <Logo isMobile={isMobile} />
+          </Link>
+
+          {/* TODO: reference: epic-react-gatsby-main/src/layouts/video-resource-page-layout.js */}
```

Comment:
Progress subcomponent in navigation for the large viewports is not implemented yet